mikeio1d/result_network/result_global_data.py

```diff
@@ -36,8 +51,7 @@ def set_quantities(self):
 
     def add_to_result_quantity_maps(self, quantity_id, result_quantity):
         """Add global data result quantity to result quantity maps."""
-        query = QueryDataGlobal(quantity_id)
```

Comment:
Though it is only a removal of a single line, but for me it shows that introduction of TimeSeriesId and related refactoring is a very good thing :-).